### PR TITLE
Added test for signature help

### PR DESCRIPTION
**Pull request by akshita31**

### CODE REVIEW

File: test/integrationTests/signatureHelp.integration.test.ts [hunk 1]
```diff
@@ -87,6 +87,12 @@ suite(`SignatureHelp: ${testAssetWorkspace.description}`, function () {
         expect(c.signatures[0].parameters[c.activeParameter].label).to.equal(answer);
     });
 
+    test("Signature Help identifies active parameter based on comma for multpilte commas", async function () {
```
Comment (TheRealPiotrP) [inline test/integrationTests/signatureHelp.integration.test.ts]: multpilte -> multiple

File: test/integrationTests/signatureHelp.integration.test.ts [hunk 2]
```diff
@@ -87,6 +87,12 @@ suite(`SignatureHelp: ${testAssetWorkspace.description}`, function () {
...
+        let c = <vscode.SignatureHelp>await vscode.commands.executeCommand("vscode.executeSignatureHelpProvider", fileUri, new vscode.Position(18, 25));
+        let answer = `double double1`;
```
Comment (TheRealPiotrP) [inline test/integrationTests/signatureHelp.integration.test.ts]: I'd inline this value on the next line. If you prefer it to be separate, I'd rename the variable to `expected`

File: test/integrationTests/signatureHelp.integration.test.ts [hunk 3]
```diff
@@ -87,6 +87,12 @@ suite(`SignatureHelp: ${testAssetWorkspace.description}`, function () {
         expect(c.signatures[0].parameters[c.activeParameter].label).to.equal(answer);
     });
 
+    test("Signature Help identifies active parameter based on comma for multpilte commas", async function () {
+        let c = <vscode.SignatureHelp>await vscode.commands.executeCommand("vscode.executeSignatureHelpProvider", fileUri, new vscode.Position(18, 25));
```
Comment (rchande) [inline test/integrationTests/signatureHelp.integration.test.ts]: Bonus points: replace the many instances of ` = <vscode.SignatureHelp>await vscode.commands.executeCommand("vscode.executeSignatureHelpProvider", fileUri, new vscode.Position(` with calls to a function that takes a fileuri and position and returns a vscode.SignatureHelp.

File: test/integrationTests/signatureHelp.integration.test.ts (hunk 2 above)
Comment (rchande) [reply test/integrationTests/signatureHelp.integration.test.ts]: +1

**Pull request merged**